cli/utils/prepare-library-package.js

```diff
@@ -18,15 +18,22 @@ function makePackageFileForDist() {
 }
 
 function copyFilesToDist() {
-  fs.copySync(
-    skyPagesConfigUtil.spaPath('README.md'),
-    skyPagesConfigUtil.spaPath('dist', 'README.md')
-  );
+  try {
```

Comment:
I don't like wrapping this in a try/catch, especially if we're not going to do anything with the caught `err`.

Reply:
I wrapped it in a `try` so that if the library doesn't include these files (readme, changelog, or assets) it will not fail (due to how `fs` throws errors). I thought it was better than writing out "does directory exist" conditions. If you'd rather have those, I can add them.

Reply:
The `try/catch` would make it ambiguous as to whether there was a real problem copying the files versus if they files didn't exist to copy.  I think I would prefer to not use it.  I'll let you decide if it's worth optimizing now we have three almost identical blocks of code.

Reply:
Sounds good; I'll take a second swing at it!